tests/TestCoreModelExtension.php

```diff
@@ -1,197 +1,61 @@
 <?php
 
-use Astrotomic\Translatable\Test\Model;
-use Astrotomic\Translatable\Test\Model\City;
-use Astrotomic\Translatable\Test\Model\Company;
-use Astrotomic\Translatable\Test\Model\Country;
-use Astrotomic\Translatable\Test\Model\Continent;
 use Astrotomic\Translatable\Test\Model\Vegetable;
-use Astrotomic\Translatable\Test\Model\CountryStrict;
-use Astrotomic\Translatable\Test\Model\CountryGuarded;
-use Astrotomic\Translatable\Test\Model\CityTranslation;
-use Astrotomic\Translatable\Test\Model\CountryTranslation;
 
-final class TestCoreModelExtension extends TestsBase
+final class TestCoreModelExtension extends TestCase
 {
-    // Saving
-
-    public function test_it_saves_empty_instances()
+    
+    /** @test */
+    public function to_array_returns_translated_attributes()
     {
-        $company = new Company();
-        $company->save();
-        $this->assertGreaterThan(0, $company->id);
+        $vegetable = factory(Vegetable::class)->make(['name:en' => 'Peas']);
 
-        $country = new Continent();
-        $country->save();
-        $this->assertGreaterThan(0, $country->id);
+        $this->assertArrayHasKey('name', $vegetable->toArray());
```

Comment:
Could be extended by a second or single assertion `assertEquals('Peas', $vegetable->toArray()['name'])`. Same as in `translated_attributes_can_be_accessed_as_properties`